content/docs/1.17/operator.md

```diff
@@ -263,12 +263,13 @@ spec:
           topic: jaeger-spans
           brokers: my-cluster-kafka-brokers.kafka:9092
       ingester:
-        deadlockInterval: 5 # <2>
+        deadlockInterval: 5s # <2>
   storage:
     type: elasticsearch
     options:
       es:
         server-urls: http://elasticsearch:9200
+        index-prefix: my-prefix
```

Comment:
Not sure why this `index-prefix` parameter should be included here? The example is primarily about streaming, not the storage. So think this one should be removed.

Reply:
I moved this to the External Elasticsearch part.